lib/TPP/GPU/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+add_mlir_library(TPPGPU
```

Comment:
I would assume that everything is TPP-centric. The current GPU infrastructure is pretty generic but it will evolve to be more tailor-made for our specific needs.
It is just a subdirectory to keep every GPU related in one place for convenience.